airflow-core/src/airflow/ui/src/pages/Dashboard/PoolSummary/PoolSummary.tsx

```diff
@@ -0,0 +1,113 @@
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, Heading, Flex, Skeleton } from "@chakra-ui/react";
+import { BiTargetLock } from "react-icons/bi";
+
+import { usePoolServiceGetPools } from "openapi/queries/queries";
+import { Tooltip } from "src/components/ui";
+import { capitalize } from "src/utils";
+import { slots } from "src/utils/slots";
+
+export const PoolSummary = () => {
+  const { data, isLoading } = usePoolServiceGetPools();
+  const pools = data?.pools ?? [];
+  const totalSlots = pools.reduce((sum, pool) => sum + pool.slots, 0);
+
+  const slotTotals = {
+    deferred_slots: 0,
+    occupied_slots: 0,
+    open_slots: 0,
+    queued_slots: 0,
+    running_slots: 0,
+    scheduled_slots: 0,
+  };
+
+  const poolsWithSlotType = {
+    deferred_slots: 0,
+    occupied_slots: 0,
+    open_slots: 0,
+    queued_slots: 0,
+    running_slots: 0,
+    scheduled_slots: 0,
+  };
+
+  pools.forEach((pool) => {
+    Object.keys(slotTotals).forEach((slotKey) => {
+      const typedKey = slotKey as keyof typeof slotTotals;
+      const slotValue = pool[typedKey];
+
+      if (typeof slotValue === "number" && slotValue > 0) {
+        slotTotals[typedKey] += slotValue;
+        poolsWithSlotType[typedKey] += 1;
+      }
+    });
+  });
+
+  return (
+    <Box w="100%">
+      <Flex color="fg.muted" mb={2} w="100%">
+        <BiTargetLock />
+        <Heading ml={1} size="xs">
+          Pool Slots
+        </Heading>
+      </Flex>
+
+      {isLoading ? (
+        <Skeleton borderRadius="full" h={8} w="100%" />
+      ) : (
+        <Box>
+          <Flex bg="white" borderRadius="full" overflow="hidden" w="100%">
+            {Object.entries(slots).map(([slotKey, { color, icon }]) => {
+              const typedKey = slotKey as keyof typeof slotTotals;
+              const slotValue = slotTotals[typedKey];
+              const flexValue = slotValue / totalSlots || 0;
+
+              if (flexValue === 0) {
+                return undefined;
+              }
+
+              const tooltipContent = `${capitalize(slotKey.replace("_", " "))}: ${slotValue} ${
+                poolsWithSlotType[typedKey] > 1 ? `(${poolsWithSlotType[typedKey]} pools)` : ""
+              }`;
+
+              return (
+                <Tooltip content={tooltipContent} key={slotKey}>
+                  <Flex
+                    alignItems="center"
+                    bg={`${color}.solid`}
+                    color="white"
+                    flex={flexValue}
+                    gap={1}
+                    h="100%"
+                    justifyContent="center"
+                    py={0.5}
+                    textAlign="center"
+                  >
+                    {icon}
+                    {slotValue}
+                  </Flex>
+                </Tooltip>
+              );
+            })}
+          </Flex>
```

Comment:
I think we should refactor and reuse here the `PoolBar` component. (make it a shared component. 

It's weird to have a slightly different design between the two. (border radius, icons vs number etc...)